### ISSUE TITLE
Some translation problems

### ISSUE THREAD

**Issue by Anzure** [drtshock/Factions#734]
**These messages are missing or partly missing from the translation:**
- Your faction already owns this land
- Only faction admins can Gi eller fjern Offiser rettigheter
- Faction admin Eirik has been removed. Laurern has been promotet as the new faction admin.
- Only faction moderators can sparke ut en spiller fra Factionen.
- Only faction moderators can Change the faction tag
- Du har ikke tillatelse til å designate a faction as peaceful
- Note:/f unclaim,declaim works on safe/war zones as well
- https://gyazo.com/018c0ab59b8cf701033bc87f933f870d (minute and seconds is not possible to translate)

As you can see half of the messages are translated, the rest cannot be translated. I suggest you use Notepad++ and search folders using that.

**Grammar problems:**
COMMAND.ADMIN.PROMTED needs to be divided into 3 messages in the lang file:
- You did [player] to Leader of [faction].
- [player] made you Leader of your Faction.
- [player] made [player] leader of [faction].

Basically you can't use "You" as a variable for usernames, it screws up the grammar. You see we got multiple words for "you" in Norwegian, and that's pretty normal in other languages as well.
I hope you somehow understand even though you only speak English. COMMAND.ADMIN.PROMTED is not the only one, but it's the most impossible one to get correct.

**Comment by Anzure** [comment]
Bump

**Comment by Anzure** [comment]
Bump

**Comment by Malachiel87** [comment]
Please fix these messages, after 2 years ... bump xD
